MurungaKibaara/Questioner
# Ch create directory structure 163063677

*What does this PR do?*

This pull request creates the recommended app structure

*Description of tasks to be completed*

The following are the tasks to be completed  
Create the app structure

*How should you manually test this?*

Clone this repository  
`$ git clone https://github.com/murungakibaara

Cd into the directory  
`$ cd Questioner'

Change into the current branch  
`$ git checkout CH-create-directory-structure-163063677`  
check the directory structure

Pivotal tracker story associated with this PR:  
#163063677

Pivotal tracker link:
https://www.pivotaltracker.com/n/projects/2235336

Screenshot

![screenshot from 2019-01-09 04-04-18](https://user-images.githubusercontent.com/43511880/50869140-d71c0900-13c3-11e9-846c-ca8b292ef5cd.png)
/Questioner.git`
